src/main/scala/com/advancedtelematic/director/http/AdminResource.scala

```diff
@@ -254,12 +252,7 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
           queueForDevice(ns, device)
         } ~
         (path("cancel") & put) {
-          val f = cancelUpdate.one(ns, device).flatMap{ res =>
-            messageBusPublisher
-              .publish(UpdateSpec(ns, device, UpdateStatus.Canceled))
-              .map(_ => res.device)
-          }
-          complete(f)
+          complete(cancelUpdate.one(ns, device).map(_.device))
```

Comment:
Should we add another `Deprecated in favor of "assignments"` comment here?

Reply:
Yeah, good point